### ISSUE TITLE
Conditional import of Keras/TFLite in model.py

### ISSUE THREAD

**Issue by fegue** [birdnet-team/BirdNET-Analyzer#6]
When running BirdNET on a Raspberry Pi it is sufficient to install `tflite-runtime` without the need to install `tensorflow`.
Not having to install tensorflow/keras keeps the installation smaller and the process simpler.

It would be great if the import could be conditional so that the script does not throw an error when there is no installation of `tensorflow`.

It could be something like
```Python
if cfg.MODEL_PATH.endswith('.tflite'):
    try:
        import tflite_runtime.interpreter as tflite
    except ModuleNotFoundError:
        from tensorflow import lite as tflite
else:
   from tensorflow import keras
```
